superset/db_engine_specs/presto.py

```diff
@@ -841,47 +843,6 @@ def _process_array_data(
                             array_value[array_child] = ""
         return all_array_data
 
-    @classmethod
-    def _consolidate_array_data_into_data(
```

Comment:
unused? @betodealmeida is this used in a cherry or something?

Reply:
A bit or archeology points to https://github.com/apache/incubator-superset/pull/7625 @khtruong , safe to remove this?